dev-client/__tests__/integration/LoginScreen-test.tsx

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright © 2024 Technology Matters
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published
+ * by the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program. If not, see https://www.gnu.org/licenses/.
+ */
+
+import {LoginScreen} from 'terraso-mobile-client/screens/LoginScreen';
+import {render} from '@testing/utils';
+
+test('renders correctly', () => {
+  expect(
```

Comment:
Suggestion: use the same pattern as the other files, getting the screen variable and doing `expect(screen).toMatchSnapshot();` at the end -- unless there's some reason for the different pattern here?

Reply:
nope! that was just the first one i wrote and i forgot to go back and update it